spec/hbw/features/tasks/search_spec.rb

```diff
@@ -0,0 +1,70 @@
+feature 'Check searching in table with tasks', js: true do
+  before(:each) do
+    user = FactoryBot.create(:user)
+
+    signin(user.email, user.password)
+    expect(page).not_to have_content 'Sign in'
+    expect(page).to     have_content 'Orders list'
+
+    ajax { click_on 'Tasks' }
+    expect_widget_presence
+    expect(page).to have_content 'Open tasks'
+  end
+
+  describe 'search field rendered' do
+    it 'with too short search query' do
+      fill_search_field('te')
+      expect_widget_presence
+    end
+
+    it 'with valid search request' do
+      expect(tasks_table_content).to eq(
+        [
+          ['Medium', 'Check test form', ' Test name', 'No description', 'expired'],
+          ['High', 'Check test form', ' Test name', 'Some test description', '30 Jun 2016']
+        ]
+      )
+
+      fill_search_field('description')
```

Comment:
Not the best example. We have placeholder `No description` which contains word `description` but we can't see this row after searching. This is a bit confusing, let's choose other query.